### PR TITLE
🔧 drop collection path

### LINKED ISSUE #9145: Link after drop minted out on landing page should go to collection instead explorer
image

now, I land at 
- https://canary.kodadot.xyz/ahp/explore/items?listed=true&collections=82

We should redirect the user to

- https://canary.kodadot.xyz/ahp/collection/82?listed=true

why
---
user have activity on the next click and see collection stats right away

### PR DESCRIPTION
**Pull request by roiLeo**
- close #9145

### CODE REVIEW

**Comment by yangwao** [comment]
nope cc @roiLeo 

image

File: components/collection/unlockable/utils/useUnlockableTag.ts [hunk 1]
```diff
@@ -33,7 +33,7 @@ export const useUnlockableTag = (small: boolean) => {
 
   const to = computed(() =>
     isMintedOut.value
-      ? `/${DEFAULT_DROP.chain}/explore/items?listed=true&collections=${DEFAULT_DROP.id}`
+      ? `/${DEFAULT_DROP.chain}/collection/{DEFAULT_DROP.id}?listed=true`
```
Comment (roiLeo) [inline components/collection/unlockable/utils/useUnlockableTag.ts:36]:
```suggestion
      ? `/${DEFAULT_DROP.chain}/collection/${DEFAULT_DROP.id}?listed=true`
```

Comment (roiLeo) [reply components/collection/unlockable/utils/useUnlockableTag.ts:36]: images

**Comment by yangwao** [comment]
Sometimes I see this bug on landing 
unrelated to this pr for sure 

image

**Comment by roiLeo** [comment]
> Sometimes I see this bug on landing unrelated to this pr for sure

If you manage to reproduce it I can take a look

**Pull request merged**